### PR TITLE
Fix: clicking video end overlay makes it playback

### PR DESCRIPTION
**Pull request by eldiegod**
Fixes: #1768

Additional quality of life change:
Made it so clicking on the next video card on the video end overlay
takes you to that video

### CODE REVIEW

File: src/components/_video/VideoPlayer/VideoOverlays/EndingOverlay.tsx [hunk 1]
```diff
@@ -103,17 +103,32 @@ export const EndingOverlay: React.FC<EndingOverlayProps> = ({
...
+      onClick={(e) => {
+        e.stopPropagation()
+      }}
+    >
       {randomNextVideo ? (
         <Container isFullScreen={isFullScreen}>
-          <InnerContainer>
+          <InnerContainer
+            onClick={() => {
+              navigate(absoluteRoutes.viewer.video(randomNextVideo.id))
```
Comment (rafalpawlow) [inline src/components/_video/VideoPlayer/VideoOverlays/EndingOverlay.tsx:115]: As above, move inline function to const

File: src/components/_video/VideoPlayer/VideoOverlays/EndingOverlay.tsx [hunk 2]
```diff
@@ -103,17 +103,32 @@ export const EndingOverlay: React.FC<EndingOverlayProps> = ({
...
             <VideoThumbnail src={thumbnailUrl} />
             <VideoInfo>
               <Text variant={mdMatch ? 't300' : 't200'} secondary>
                 Up next
               </Text>
               <Heading variant={mdMatch ? 'h500' : 'h400'}>{randomNextVideo.title}</Heading>
-              <StyledChannelLink id={channelId} avatarSize="default" textVariant={mdMatch ? 't300' : 't200'} />
+              <StyledChannelLink
+                onClick={(e) => {
+                  e.stopPropagation()
```
Comment (rafalpawlow) [inline src/components/_video/VideoPlayer/VideoOverlays/EndingOverlay.tsx]: As above, move inline function to const

File: src/components/_video/VideoPlayer/VideoOverlays/EndingOverlay.tsx [hunk 3]
```diff
@@ -103,17 +103,32 @@ export const EndingOverlay: React.FC<EndingOverlayProps> = ({
   }, [currentThumbnailUrl, randomNextVideo, randomNextVideoThumbnailUrl])
 
   return (
-    <OverlayBackground>
+    <OverlayBackground
+      onClick={(e) => {
```
Comment (rafalpawlow) [inline src/components/_video/VideoPlayer/VideoOverlays/EndingOverlay.tsx]: Let's move this function to const. Inline functions creates new reference on each render which cause unnecessary rerender.

File: src/components/_video/VideoPlayer/VideoOverlays/EndingOverlay.tsx [hunk 4]
```diff
@@ -32,7 +32,7 @@ type EndingOverlayProps = {
   isEnded: boolean
 }
 // 10 seconds
-const NEXT_VIDEO_TIMEOUT = 10000
+const NEXT_VIDEO_TIMEOUT = 10 * 1000
```
Comment (rafalpawlow) [inline src/components/_video/VideoPlayer/VideoOverlays/EndingOverlay.tsx:35]: Why did you make this change?

**Review by rafalpawlow — CHANGES_REQUESTED** [review]

File: src/components/_video/VideoPlayer/VideoOverlays/EndingOverlay.tsx (hunk 3 above)
Comment (kdembler) [reply src/components/_video/VideoPlayer/VideoOverlays/EndingOverlay.tsx]: Moving it to const doesn't really change anything, `const` will have a new reference on each render as well

Comment (rafalpawlow) [reply src/components/_video/VideoPlayer/VideoOverlays/EndingOverlay.tsx]: Yeah I was thinking about `useCallback`. But either way, we have 2 inline functions here which does the same, so maybe it's worth to declare it above?

Comment (kdembler) [reply src/components/_video/VideoPlayer/VideoOverlays/EndingOverlay.tsx]: Sure let's move it to `const` if we duplicate but let's not overuse `useCallback` if not necessary

File: src/components/_video/VideoPlayer/VideoOverlays/EndingOverlay.tsx (hunk 4 above)
Comment (eldiegod) [reply src/components/_video/VideoPlayer/VideoOverlays/EndingOverlay.tsx:35]: just to make it clearer that it is 10 secs and easier to change later

**Review by rafalpawlow — APPROVED** [review]

**Pull request merged**